tests/functional/conftest.py

```diff
@@ -33,24 +40,23 @@ def pytest_configure(config):
     # check for skipping setup on xdist master process
     if not config.pluginmanager.getplugin("dsession"):
         suffix = str(uuid.uuid4().int)
-        env, db = dr_usertool_setup()
+        env = dr_usertool_setup()
+        admin_api_key = get_admin_api_key()
 
         # User credentials
         user_username = "local-custom-model-templates-tests-{}@datarobot.com".format(suffix)
-        user_password = "lkjkljnm988989jkr5645tv_{}".format(suffix)
+        user_password = "Lkjkljnm988989jkr5645tv_{}".format(suffix)
```

Comment:
an uppercase character is required

Reply:
isn't it nice that business logic rules now are enforced for user creation ;)